pyproject.toml

```diff
@@ -40,13 +39,13 @@ select = [
 ]
 force-exclude = true
 ignore = [
-  "E402",  # module-import-not-at-top-of-file
-  "E501",  # line-too-long
-  "E741",  # ambiguous-variable-name
-  "F401",  # unused-import
-  "F402",  # import-shadowed-by-loop-var
-  "F403",  # undefined-local-with-import-star
-  "F811",  # redefined-while-unused
+  "E402", # module-import-not-at-top-of-file
+  "E501", # line-too-long
+  "E741", # ambiguous-variable-name
+  "F401", # unused-import
+  "F402", # import-shadowed-by-loop-var
+  "F403", # undefined-local-with-import-star
+  "F811", # redefined-while-unused
```

Comment:
Do we need this change?

Reply:
I think it was changed by some automatic tooling I will revert